Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -646,16 +651,12 @@ def key(pn):
 		data.append(compiledPoints[sharedPoints])
 		tupleVariationCount |= TUPLES_SHARE_POINT_NUMBERS
 
-		# Only allowed to reuse one set.
-		pointSitu = [sharedPoints if sharedPoints == points else None
-			     for points in pointSitu]
-
-		del pointSetCount
-	else:
-		pointSitu = [None] * len(variations)
+	# NotImplemented means "use shared points"
```

Comment:
using NotImplemented like that looks a bit weird to me. I think it would be clearer and need no extra comment if you added an additional `useSharedPoints: bool` parameter to compile method, and use None `if points != sharedPoints`. But take my comment as optional.

Reply:
Since the caller already has to compute the points for each TupleVariation to figure out which pointset to share, I made the `compile()` method get compiled `pointData` directly now. So there won't be any `if points != sharedPoints` possible inside since I wanted to avoid calling `getUsedPoints()` again.

I can use another singleton value if you prefer. Or I can use `b''`; still would need a comment because it's going to be a special-case.

Reply:
> Or I can use `b''`; still would need a comment because it's going to be a special-case.

Doing that now.